README.md

```diff
@@ -1,2 +1,38 @@
-# Second-proyect-Laylooo
- 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Second project
+## IMPORTAN! attribution and acknowledgment
+this proyect is the work of (https://github.com/Laylooo) she gave me permition to use it in this time for this practice.
```

Comment:
- [ ] For example of the inline comments, politely asked, and providing a 'why' for the request. 
Please make sure to check the text of your README for misspellings and grammar errors. This helps to make the repo look good and professional. Also, keep in mind that such small mistakes might leave a bad impression on the future viewers.

_To start an inline comments, go to the files changed page in the PR, choose the block of code you want to ask a request on, write the comment, then click the "start a review" button._ 🥰🙏🏻